src/service.ts

```diff
@@ -0,0 +1,11 @@
+import users from './api/users';
+import { ToDo } from './types/todo';
+import { User } from './types/user';
+
+export function findUserById(id: number): User {
+  return users.filter(user => user.id === id)[0];
```

Comment:
Consider using `find` instead of `filter` to locate the user by ID. `find` will return the first matching element and is more efficient for this use case: `return users.find(user => user.id === id);`